File: CPP/Program/Mahasiswa.cpp
#include <iostream>
#include <string>

using namespace std;

class Mahasiswa
{
private:
  string nama;
  string nim;
  string prodi;
  string fakultas;

public:
  Mahasiswa()
  {
    this->nama = "";
    this->nim = "";
    this->prodi = "";
    this->fakultas = "";
  }

  void setNama(string nama)
  {
    this->nama = nama;
  }

  string getNama()
  {
    return this->nama;
  }

  void setNim(string nim)
  {
    this->nim = nim;
  }

  string getNim()
  {
    return this->nim;
  }

  void setProdi(string prodi)
  {
    this->prodi = prodi;
  }

  string getProdi()
  {
    return this->prodi;
  }

  void setFakultas(string fakultas)
  {
    this->fakultas = fakultas;
  }

  string getFakultas()
  {
    return this->fakultas;
  }

  ~Mahasiswa(){};
};

File: Python/Program/Mahasiswa.py
class Mahasiswa():
    __nama = ""
    __nim = ""
    __prodi = ""
    __fakultas = ""

    def __init__(self, nama="", nim="", prodi="", fakultas=""):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama

    def setNim(self, nim):
        self.__nim = nim

    def getNim(self):
        return self.__nim

    def setProdi(self, prodi):
        self.__prodi = prodi

    def getProdi(self):
        return self.__prodi

    def setFakultas(self, fakultas):
        self.__fakultas = fakultas

    def getFakultas(self):
        return self.__fakultas

    def printHasil(self):
        print("Nama :" + str(self.__nama))
        print("NIM :" + str(self.__nim))
        print("Program Studi :" + str(self.__prodi))
        print("Fakultas :" + str(self.__fakultas))
